content/apps/homeassistant/contents.lr

```diff
@@ -0,0 +1,18 @@
+_model: app
+---
+title: Home-Assistant Integration
+---
+author: Fabian Affolter
```

Comment:
Are you affiliated with a hackerspace? Then you could add a `space: <yourspace>` entry :slightly_smiling_face: